src/Microsoft.ML.Dnn/DnnUtils.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.ML.Data;
 using Microsoft.ML.Runtime;
 using Tensorflow;
+using static Microsoft.ML.Dnn.ImageClassificationTrainer;
 using static Tensorflow.Binding;
 
 namespace Microsoft.ML.Transforms.Dnn
```

Comment:
(nit) This namespace seems wrong since we have `Microsoft.ML.Dnn` now .

Reply:
We have other public types in this assembly that we should re-evaluate their namespaces as well.

`Microsoft.ML.Transforms.LearningRateScheduler`.
`Microsoft.ML.Transforms.DnnModel`.

---
In reply to: [339194479](https://github.com/dotnet/machinelearning/pull/4372#discussion_r339194479) [](ancestors = 339194479)

Reply:
I think `DnnModel` should be made `internal`. I don't see it exposed anywhere.

---
In reply to: [339195151](https://github.com/dotnet/machinelearning/pull/4372#discussion_r339195151) [](ancestors = 339195151,339194479)

Reply:
Done. I was planning on doing a final API surface review and changing the namespace to Microsoft.ML.Vision in the end. 

---
In reply to: [339195669](https://github.com/dotnet/machinelearning/pull/4372#discussion_r339195669) [](ancestors = 339195669,339195151,339194479)